includes/admin/class-admin.php

```diff
@@ -22,97 +22,100 @@
 
 if ( ! class_exists( 'WPBDP_Admin' ) ) {
 
-    /**
-     * Class WPBDP_Admin
-     */
-    class WPBDP_Admin {
+	/**
```

Comment:
Let's back out the extra phpcs changes here too

Reply:
Removed extra PHPCS, but diff seems to still be big. Compared with latest version from master and only added tabs on changed files